### DISCUSSION TITLE
How to custom prometheus to not install kubeRbacProxy

### DISCUSSION THREAD

**Discussion by Icedroid** [prometheus-operator/kube-prometheus#567] [General]
Could I custom not install kubeRbacProxy by jsonet ? Because I want to visit the node-exporter endpoint from outside of cluster

**Comment by brancz** [comment]
1) you could still get a serviceaccount token from the outside
2) you can use jsonnet to remove the container in your setup, be aware that that exposes potentially sensitive data about your nodes (such as kernel version, which is valuable information to exploit :slightly_smiling_face:, which is why I do not recommend leaving your node-exporter insecure)

**Comment by Icedroid** [comment]
use jsonnet to remove the container in your setup, how to do?

**Comment by brancz** [comment]
You can use the `std.filter` function to filter out that container.
